assets/tuned/08-ds-tuned.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: tuned
+  namespace: openshift-cluster-node-tuning
+  labels:
+    openshift-app: tuned
+spec:
+  selector:
+    matchLabels:
+      openshift-app: tuned
+  template:
+    metadata:
+      labels:
+        openshift-app: tuned
+    spec:
+      serviceAccountName: tuned
+      automountServiceAccountToken: true
+      containers:
+      - command:
+        - /var/lib/tuned/bin/run
+        image: docker.io/jmencak/tuned-poc
```

Comment:
do you need to request anything from RCM to make this real?

Reply:
Definitely, on my TODO list.